### DISCUSSION TITLE
Figure caption formatting

### DISCUSSION THREAD

**Discussion by katycroffbell** [knowledgefutures/pubpub#2127] [Support]
Is it possible to force figure captions to flush left? Or must they always be centered?

**Comment by katycroffbell** [comment]
Related to figures - the captions in the downloaded pdfs look...not great. Is it possible to edit the code that generates the pdf so that the captions are the same font as the body text, and smaller?

<img width="428" alt="image" src="https://user-images.githubusercontent.com/92737253/178643374-268b6932-5215-44d3-8ab7-c41d1f4bd206.png">

**Reply by gabestein** [reply]
Yeah, this is weird! Can you send me a link to the pub itself so I can take a look?

PDF css editing is not in our plans in the short-term, mostly because it's both hard to test and very easy to introduce bugs that crash the generator. But as the toolkit we use to generate it (pagedjs) matures, we are thinking about how to allow for it.

**Reply by katycroffbell** [reply]
Here you go! https://deepseacapacity.pubpub.org/pub/2022-global-regions-results

On the pub itself, the caption size looks good, but when you download the pdf it looks gigantic in comparison to body text. I don't know much about css editing, but given that the captions are a different font, color, and size than the body text, it seems as if the caption font size could just get knocked down a couple of points, and it would look much better in the pdf.

Since my initial post about this, i discovered that the caption font is dejavu sans, & the body text is dejavu serif, same as the pub, so that's cool, it's the caption size that really is the issue.

**Reply by gabestein** [reply]
Thanks! We made some updates to the way we render PDFs that gives us a lot more control over fonts and such, but it probably led to the sizing issue. Will take a look.

**Comment by gabestein** [comment]
Alignment for captions has come up before and is something we should support. Will bring it up to the team at the next sprint meeting.
